config/config.go

```diff
@@ -70,13 +70,37 @@ func DefaultConfig() *Config {
 	return &Config{Hazelcast: hz}
 }
 
-func writeToFile(config *Config, confPath string) error {
-	var err error
-	var out []byte
-	if out, err = yaml.Marshal(config); err != nil {
-		return err
-	}
-	return file.CreateMissingDirsAndFileWithRWPerms(confPath, out)
+const defaultUserConfig = `hazelcast:
+  clientname: ""
+  logger:
+    level: error
+  cluster:
+    security:
+      credentials:
+        username: ""
+        password: ""
+    name: dev
+    cloud:
+      token: ""
+      enabled: false
+    discovery:
+      usepublicip: false
+    unisocket: true
+  network:
+      addresses: []
+      connectiontimeout: 0s
```

Comment:
I think we need a comment on this config item that explains `0s == infinity`

Reply:
I added a comment but not sure if it is ok. I'd love to hear your suggestions if you have any https://github.com/hazelcast/hazelcast-commandline-client/pull/103/commits/e36873bca1e2ed61955ab506ce8f43f9152267a7